defaults.properties.yml

```diff
@@ -1,30 +1,192 @@
-build:
-  artifact_mode: symlink
```

Comment:
I think that the only place that it really makes sense to copy instead of symlink is when building the artifact, which is done inline -- so I thought that relying on the class behavior/defaults makes sense instead of treating this as a configuration property.

Reply:
Sounds good to me.  That helps to simplify things a bit.